README.md

```diff
@@ -0,0 +1,96 @@
+# Complete integration sample for the Corbado web component
+This is a sample implementation of frontend and backend where the Corbado web component is integrated.
+
+## 1. File structure
+    .
+    ├── ...
+    ├── config                        
+    │   └── routes.yaml                 # Assigns paths to controller methods    
+    ├── src                             
+    │   ├── Controller                  
+    │   │   ├── BackendController.php   # Manages endpoints for backend
+    │   └── └── FrontendController.php  # Manages endpoints for frontend
+    ├── templates                     
+    │   ├── home.html.twig              # Home page which you only get to see if you are logged in
+    │   └── login.html.twig             # Login page which contains the Corbado web component
+    └── ...
+
+## 2. How to use
+>**Warning**
+>This sample code corresponds to the integration tutorial in our docs, please read it first in order to understand the flows and business logic!
+
+### 2.1. Prerequisites
+The only thing you need is a CNAME which points to `auth.corbado.com`. We will use `auth.your-company.com` in this tutorial. More info on what a CNAME is and why it is needed can be found in our [docs](https://docs.corbado.com/integrations/web-component#1.-define-cname).
+
+### 2.2. Corbado developer panel settings
+
+>**Warning**
+>Remember to always press "Save changes" after entering details in the Corbado developer panel!
+>
+#### 2.2.1. Configure CNAME
+
+In the developer panel under `Project settings -> Web component` enter the CNAME you previously created. 
+![image](https://user-images.githubusercontent.com/23581140/205950309-f6f622e5-94ca-4413-9384-d7a2605da75d.png)
+
+#### 2.2.2. Fill in your backend endpoints
+
+Under `Project settings -> Web component` configure the endpoints as seen in the following image, but use your own ngrok url (the paths stay the same).
+```
+<your-ngrok-url>/api/sessionToken
+<your-ngrok-url>/api/loginInfo
+<your-ngrok-url>/api/passwordVerify
+```
```

Comment:
Hier nochmal erwähnen, was in unserem Fall konkret die ngrok URL ist. Bislang sieht man das nur im Screenshot